tests/unit/dump.tcl

```diff
@@ -127,20 +127,18 @@ start_server {tags {"dump"}} {
     test {RESTORE key with future RDB version, strict version check} {
         #              str len    RDB 222 CRC64 checksum
         #               |   |      |       |
-        set bar_dump "\x00\x03bar\xde\x00\x0fYUza\xd3\xec\xe0"
-       assert_error {ERR DUMP payload version or checksum are wrong} {r restore foo 0 $bar_dump replace}
+        set bar_dump "\x00\x03\x62\x61\x72\xde\x00\x0fYUza\xd3\xec\xe0"
+        assert_error {ERR DUMP payload version or checksum are wrong} {r restore foo 0 $bar_dump replace}
     }
 
     test {RESTORE key with future RDB version, relaxed version check} {
         #              str len    RDB 222 CRC64 checksum
         #               |   |      |       |
-        set bar_dump "\x00\x03bar\xde\x00\x0fYUza\xd3\xec\xe0"
-        r config set rdb-version-check relaxed
-        catch {r restore foo 0 $bar_dump replace} e
+        r config set rdb-version-check relaxed 
+        r restore foo 0 "\x00\x03\x62\x61\x72\xde\x00\x0fYUza\xd3\xec\xe0" replace
```

Comment:
I suggest swapping the line 137 and 138, and fix the comment order as well,

Reply:
The `config set` needs to be before `restore`, but we can move the comment to here and fix the alignment.

Is this comment style better?

```suggestion
        #               |type|len|           |  RDB  |      CRC64        |
        #               |str | 3 |   "bar"   |  222  |     checksum      |
        r restore foo 0 "\x00\x03\x62\x61\x72\xde\x00\x0fYUza\xd3\xec\xe0" replace
```